cpp/engine.cpp

```diff
@@ -170,9 +170,16 @@ Engine::Engine(util::Dir *data_dir, const char *windowtitle)
 	if (devices.empty()) {
 		throw util::Error{MSG(err) << "No audio devices found"};
 	}
+
+	// initialize profiler
+	this->profiler.register_category(util::Profiler::Categorie::EVENT_PROCESSING);
+	this->profiler.register_category(util::Profiler::Categorie::RENDERING);
+	this->profiler.register_category(util::Profiler::Categorie::IDLE_TIME);
```

Comment:
Yeah, I was thinking about moving the register part into the start_measure method and register the categorie only if it is not already registered